LockPatron/Controllers/WeatherForecastController.cs

```diff
@@ -35,6 +35,7 @@ public IQueryable<WeatherForecast> Get([FromQuery, Range(0, 50)] int count)
     [HttpPost]
     public async Task<WeatherForecast> Post()
     {
+      _logger.LogTrace("Adding new forecast.");
```

Comment:
It looks like some other code got added in the commit.

Reply:
Yep, was an old commit, now it's pruned.